src/silicone/stats.py

```diff
@@ -103,37 +104,122 @@ def rolling_window_find_quantiles(
     return results
 
 
-def calc_quantiles_of_data(distribution, points_to_quant):
+def calc_quantiles_of_data(
+    distribution, points_to_quant, smoothing=None, weighting=None, to_quantile=True
+):
     """
     Calculates the quantiles of points_to_quant in the distribution of values described
-    by distribution
+    by distribution. Optionally treats points_to_quant as quantiles and returns the
+    values that would lead to them instead.
+
     Parameters
     ----------
     distribution : pd.Series
         The distribution of values.
     points_to_quant : pd.Series
-        The points which we want to know the quantiles of.
+        The points which we want find: if to_quantile == True (default) these are the
+        values which we will compare to the distribution, if false, these are the
+        quantiles which we want to find.
+    smoothing : float or string
+        By default, no smoothing is done on the distribution. If an argument is used,
+        it is fed into scipy.stats.gaussian_kde. If a float is input, we
+        fit a Gaussian kernel density estimator with that width to the points and return
+        the quantiles of that distribution. If a string is used, it must be either
```

Comment:
```suggestion
        . If a string is used, it must be either
```

I'd remove the stuff about what we return, that's covered elsewhere?

Reply:
ping